scripts/include/common.sh

```diff
@@ -19,9 +19,8 @@ export DOCKER_ORG="${DOCKER_ORG:-cap}"
 export DOCKER_USERNAME="${DOCKER_USERNAME:-}"
 export DOCKER_PASSWORD="${DOCKER_PASSWORD:-}"
 export ENABLE_EIRINI="${ENABLE_EIRINI:-true}"
-export CREATE_EIRINI_NAMESPACE="${CREATE_EIRINI_NAMESPACE:-false}"
 
 
 if [ -z "${DEFAULT_STACK}" ]; then
-  export DEFAULT_STACK=$(helm inspect helm/cf/ | grep DEFAULT_STACK | sed  's~DEFAULT_STACK:~~g' | sed 's~"~~g' | sed 's~\s~~g')
-fi
\ No newline at end of file
+  export DEFAULT_STACK=$(helm inspect suse/cf | grep DEFAULT_STACK | sed  's~DEFAULT_STACK:~~g' | sed 's~"~~g' | sed 's~\s~~g')
```

Comment:
Not sure about this, in this way we get the default stack only if the release was deployed in the cluster, right? ( before we were inspecting the local chart, so it could have been resolved without a deployed release)